scripts/fix_tokenization.py

```diff
@@ -72,6 +102,23 @@ def insert_retokenized(terminal, preterminal_parent, tokens, index_in_pretermina
     insert_punct(insert_index + 1, preterminal_parent, state, tokens[non_punct_index + 1:])
 
 
+def insert_retokenized_currency(i, terminals, preterminals,
+                                preterminal_parents,
+                                tokens, state):
+    if len(tokens) == 2 and tokens[0] in CURRENCIES and \
+            (tokens[1].replace('.', '', 1).isdigit() or
+             tokens[1].replace(',', '')):
+        terminals[i].text = tokens[1]
+        index_to_insert = preterminal_parents[i].getchildren(). \
+            index(preterminals[i])
+        preterminal_parents[i].insert(index_to_insert,
+                                      create_token_element(state,
+                                                           tokens[0],
+                                                           False))
```

Comment:
I think it's better to write `is_punct=False` instead of just `False` for clarity. Same for the similar cases below.